src/index.html

```diff
@@ -5,8 +5,389 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header
+    class="top-bar"
+    id="home"
+    >
+      <a href="#home">
+        <img
+        class="logo"
+        src="./images/logo.png"
+        alt="logothype"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+            href="#Laptops&Computers"
+            class="nav__link"
+            data-qa="nav-hover">
+            Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+            href="#gadgets"
+            class="nav__link">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+
+    <main class="main">
+      <section class="catalog">
+        <article class="card" data-qa="card">
+          <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="product image">
+
+          <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__review-box">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="reviews">Reviews: 5</p>
+          </div>
+
+          <div class="card__price-box">
+            <span class="card__price">Price:</span>
+
+            <div class="card__value">$2,199</div>
+          </div>
+
+          <a
+            class="card__buy"
+            data-qa="card-hover"
+            href="#"
+            >
+          Buy
+          </a>
+        </article>
+
+
+
+        <article class="card">
```

Comment:
no needs to make such a big distance
```suggestion
        </article>

        <article class="card">
```